src/components/Organisms/TodoCard/index.jsx

```diff
@@ -7,6 +7,22 @@ import Task from "../../Molecules/Task/index.jsx";
 const TodoCard = () => {
   const [taskList, settaskList] = useState([]);
 
+  useEffect(() => {
+    const data = localStorage.getItem("taskList");
+    if (data === null) {
+      return;
+    } else {
+      const storageData = JSON.parse(data);
+      settaskList(storageData);
+      return;
+    }
```

Comment:
```suggestion
    const strageData = localStorage.getItem("taskList");
    if (data === null) {
      return;
    } else {
      const parsedData = JSON.parse(strageData);
      settaskList(parsedData);
    }
```
個人的には↑のような変数名の方が処理が追いやすくなる気がします。
変数にどんな値が入るのかが、変数名を見ただけで分かるような命名が理想です！

Reply:
ありがとうございます